src/post/Footer.tsx

```diff
@@ -101,96 +196,40 @@ const Footer: React.FC<FooterProps> = ({
   const answerable = allowsComment || allowsContribution;
   return (
     <div className="footer" ref={ref}>
-      <div
-        className={classnames("notes", {
-          "with-updates": newContributions || newComments,
-        })}
-      >
-        {(!!newContributions || !!newComments) && (
-          <div className="notes-update">
-            {!!newContributions && (
-              <span className="contributions-update">
-                {newContributions > 99 ? "∞" : newContributions}
-                <FontAwesomeIcon icon={faPlusSquare} />
-              </span>
-            )}
-            {!!newComments && (
-              <span className="comments-update">
-                {newComments > 99 ? "∞" : newComments}
-                <FontAwesomeIcon icon={faComment} />
-              </span>
-            )}
-          </div>
-        )}
-        <div className="notes-button">
-          <NotesDisplay
-            link={notesLink}
-            directContributions={directContributions || 0}
-            totalContributions={totalContributions || 0}
-            totalComments={totalComments || 0}
-          />
+      <NotesDisplay
+        link={notesLink}
+        directContributions={directContributions}
+        totalContributions={totalContributions}
+        totalComments={totalComments}
+        newComments={newComments}
+        newContributions={newContributions}
+        compact={currentBreakpoint == "compact"}
+      />
+      {answerable && (
+        <div className={classnames("footer-actions")}>
+          <Button
+            onClick={onContribution}
+            icon={faPlusSquare}
+            compact={currentBreakpoint == "compact"}
+            theme={ButtonStyle.TRANSPARENT}
+            disabled={!allowsContribution}
+            label="new contribution"
+          >
+            Contribute
+          </Button>
+          <Button
+            onClick={onComment}
+            icon={faComment}
+            compact={currentBreakpoint == "compact"}
+            theme={ButtonStyle.TRANSPARENT}
+            disabled={!allowsComment}
+            label="new comment"
```

Comment:
I changed them, but I wish we had an accessibility expert to ask. I feel like the visuals make it clearer what the action is, and without those it might be harder unless you're told that it's for a "new comment/contribution". But I don't know enough to be sure.

Reply:
Since they are buttons, screenreaders are supposed to announce that them as "{label} button" and I'm pretty sure you're supposed to favor verbs for labeling them. I *think* "contribute button" and "comment button" are about as clear auditorily as they are visibly, but if further clarity was needed I think "add contribution/comment" would be preferable to "new contribution/comment". But, best practices is actually for them to be labelled by the visual text content, so that's why I think they should match it, since it does exist at the larger screen sizes.

Reply:
Thank you for the explanation, I learned things! It should be good as it is then :)